scripts/preloadHTTPSE.js

```diff
@@ -33,84 +32,40 @@ const downloadRulesets = (dir, cb) => {
 }
 
 const buildDataFiles = () => {
-  // Manually exclude sites that are broken until they are fixed in the next
-  // HTTPS Everywhere release.
-  const exclusions = {
```

Comment:
we don't want to delete these exclusions unless all of these rulesets are verified to be fixed in HTTPS Everywhere upstream